src/bc.F

```diff
@@ -2679,17 +2657,15 @@ subroutine restrict_openbc_sn(ruh,rmh,rho0,v3d)
       double precision :: fluxout,fluxin,tem,u1,t3
       double precision, dimension(nk) :: temout,temin
 
-!$omp parallel do default(shared)   &
-!$omp private(k)
+!$omp parallel do default(shared) private(k)
```

Comment:
@sjsprecious, This routine does not appear to be OpenACC enabled.  Have you noticed this?  Is it in other modifications?

Reply:
This subroutine is called from the GPU region of the code.  So this is likey a bug.